app/models/project.rb

```diff
@@ -104,6 +104,13 @@ class Project < ApplicationRecord
 
   has_many :recurring_meetings, dependent: :destroy
 
+  belongs_to :template, class_name: "Project", optional: true
+  has_many :templated_projects,
+           class_name: "Project",
+           foreign_key: "template_id",
+           inverse_of: :template,
+           dependent: nil
+
```

Comment:
I am wondering whether there are - or will be - precautions for race conditions in the wizard. When you select a template, but then, while completing the form, the template is deleted 🤔